README.md

```diff
@@ -57,51 +57,34 @@ module.exports = x => Promise.resolve(x ** 2);
 
 ### Streams (experimental)
 
-Streaming functions can be created by setting the `$interactionModel` property on the function to `node-streams`.
-The function will then be invoked with two arguments, an `input` [Readable Stream](https://nodejs.org/dist/latest-v8.x/docs/api/stream.html#stream_class_stream_readable) and an `output` [Writeable Stream](https://nodejs.org/dist/latest-v8.x/docs/api/stream.html#stream_class_stream_writable).
-Both streams are object streams. Any value returned by the function is ignored, new messages must be written to the output stream.
+#### Proto generation
 
-```js
-// echo.js
-module.exports = (input, output) => {
-  input.pipe(output);
-};
-module.exports.$interactionModel = "node-streams";
+```shell
+ $ npm install -g grpc-tools
```

Comment:
That's what I wanted to do at first, but `grpc-tools` needs to be exported in the `PATH`, so that `grpc_tools_node_protoc` can use it (see `generate.sh` file in the PR).

Reply:
move generate.sh to be an npm script and it will be on the path.

Reply:
👍